README.md

```diff
@@ -1,6 +1,8 @@
 # Job Manager
 
-[![CircleCI](https://circleci.com/gh/DataBiosphere/job-manager/tree/master.svg?style=svg)](https://circleci.com/gh/DataBiosphere/job-manager/tree/master)
+![CircleCI branch](https://img.shields.io/circleci/project/github/DataBiosphere/job-manager/master.svg?label=Tests%20on%20Circle%20CI&style=flat-square)
+![Github](https://img.shields.io/badge/Supported%20Backends-cromwell%20%7C%20dsub-blue.svg?&style=flat-square)
```

Comment:
This one I'm not so sure about (eg what does it give us?)

Reply:
Oh so this one does nothing but shows that JMUI is supporting both Cromwell and dsub... not sure if this has changed 🤔